ufcc/default.ini

```diff
@@ -0,0 +1,9 @@
+[Parameters]
+cutoff = 7
```

Comment:
Add 1 or 2 lines of documentation for each parameter (what it is, how it is used).